beacon-light-client/plonky2/circuits_executables/bin/balance_verification_circuit_data_generation.rs

```diff
@@ -59,6 +59,11 @@ pub async fn async_main() -> Result<()> {
         _phantom: PhantomData::<PoseidonGoldilocksConfig>,
     };
 
+    if level != None && level.unwrap() > 37 {
+        eprintln!("\x1b[31mError: Supplied level {} is larger than the maximum allowed level 38\x1b[0m", level.unwrap());
```

Comment:
Oh right, I originally wrote the error message as being 38, because I was under the impression there were 38. But then I was told that there should only be 37, guess I forgot to edit the error message.